examples/separate_instance_rule/main.tf

```
@@ -0,0 +1,64 @@
+terraform {
+  required_providers {
+    nifcloud = {
+      source = "nifcloud/nifcloud"
+    }
+  }
+}
+
```

I want to match it with other examples, so adding a region seems like a good idea.

```suggestion

provider "nifcloud" {
  region = "jp-east-1"
}

```

---

Fixed!